### ISSUE TITLE
'schema::Migration' has no link or property 'generated_by' on migration create

### ISSUE THREAD

**Issue by fmoor** [edgedb/edgedb-cli#879]
- EdgeDB Version: 2.6+ba1661b
- EdgeDB CLI Version: 2.2.0-dev.868+fc9f7ca
- OS Version: ubuntu 22.04

Steps to Reproduce:

1. create a new project
2. run `edgedb migration create` 

output:
```
$ edgedb migration create
edgedb error: InvalidReferenceError: object type 'schema::Migration' has no link or property 'generated_by'
  Server traceback:
      Traceback (most recent call last):
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/server/compiler_pool/worker_proc.py", line 55, in worker
          res = meth(*args)
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/server/compiler_pool/worker.py", line 160, in compile
          units, cstate = COMPILER.compile(
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/server/compiler/compiler.py", line 2277, in compile
          unit_group = self._compile(ctx=ctx, source=source)
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/server/compiler/compiler.py", line 1798, in _compile
          return self._try_compile(ctx=ctx, source=source)
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/server/compiler/compiler.py", line 1862, in _try_compile
          comp, capabilities = self._compile_dispatch_ql(
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/server/compiler/compiler.py", line 1772, in _compile_dispatch_ql
          query = self._compile_ql_query(ctx, ql, script_info=script_info)
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/server/compiler/compiler.py", line 662, in _compile_ql_query
          ir = qlcompiler.compile_ast_to_ir(
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/edgeql/compiler/__init__.py", line 179, in wrapper
          return func(*args, **kwargs)
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/edgeql/compiler/__init__.py", line 237, in compile_ast_to_ir
          ir_set = dispatch_mod.compile(tree, ctx=ctx)
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/functools.py", line 889, in wrapper
          return dispatch(args[0].__class__)(*args, **kw)
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/edgeql/compiler/stmt.py", line 127, in compile_SelectQuery
          stmt.result = compile_result_clause(
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/edgeql/compiler/stmt.py", line 1300, in compile_result_clause
          expr = dispatch.compile(result_expr, ctx=ectx)
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/functools.py", line 889, in wrapper
          return dispatch(args[0].__class__)(*args, **kw)
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/edgeql/compiler/func.py", line 102, in compile_FunctionCall
          args, kwargs = compile_func_call_args(
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/edgeql/compiler/func.py", line 685, in compile_func_call_args
          arg_ir = polyres.compile_arg(arg, typemods[ai], ctx=ctx)
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/edgeql/compiler/polyres.py", line 583, in compile_arg
          arg_ir = dispatch.compile(arg_ql, ctx=argctx)
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/functools.py", line 889, in wrapper
          return dispatch(args[0].__class__)(*args, **kw)
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/edgeql/compiler/stmt.py", line 127, in compile_SelectQuery
          stmt.result = compile_result_clause(
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/edgeql/compiler/stmt.py", line 1300, in compile_result_clause
          expr = dispatch.compile(result_expr, ctx=ectx)
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/functools.py", line 889, in wrapper
          return dispatch(args[0].__class__)(*args, **kw)
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/edgeql/compiler/stmt.py", line 136, in compile_SelectQuery
          clauses.compile_where_clause(
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/edgeql/compiler/clauses.py", line 56, in compile_where_clause
          ir_expr = dispatch.compile(where, ctx=subctx)
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/functools.py", line 889, in wrapper
          return dispatch(args[0].__class__)(*args, **kw)
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/edgeql/compiler/expr.py", line 130, in compile_BinOp
          op_node = func.compile_operator(
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/edgeql/compiler/func.py", line 322, in compile_operator
          arg_ir = polyres.compile_arg(
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/edgeql/compiler/polyres.py", line 583, in compile_arg
          arg_ir = dispatch.compile(arg_ql, ctx=argctx)
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/functools.py", line 889, in wrapper
          return dispatch(args[0].__class__)(*args, **kw)
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/edgeql/compiler/expr.py", line 75, in compile_Path
          return stmt.maybe_add_view(setgen.compile_path(expr, ctx=ctx), ctx=ctx)
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/edgeql/compiler/setgen.py", line 454, in compile_path
          path_tip = ptr_step_set(
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/edgeql/compiler/setgen.py", line 616, in ptr_step_set
          ptrcls = resolve_ptr(
        File "/home/fmoor/.local/share/edgedb/portable/2.6/lib/python3.10/site-packages/edb/edgeql/compiler/setgen.py", line 771, in resolve_ptr
          raise err
      edb.errors.InvalidReferenceError: object type 'schema::Migration' has no link or property 'generated_by'
```

**Comment by msullivan** [comment]
Looks like the issue is the interaction between a 2.x edgedb that doesn't have `generated_by` and a dev branch CLI. CLI should do a version check probably

**Comment by jerber** [comment]
This is happening to me too after I upgraded the CLI. Is there a way to downgrade the CLI?

**Comment by jerber** [comment]
> This is happening to me too after I upgraded the CLI. Is there a way to downgrade the CLI?

edgedb cli upgrade --to-stable does the trick

**Comment by tailhook** [comment]
It should be fixed by #877
